### PR TITLE
Fix the deployment status condition

### PR DESCRIPTION
**Pull request by danidoni**

### CODE REVIEW

File: deploy_without_migration.yml [hunk 1]
```diff
@@ -12,4 +12,4 @@
               name: refresh_repositories
           - include_role: 
               name: deploy
-        when: deployment_allowed is succeeded
+        when: deployment_allowed.stdout.find('status') == "unlocked"
```
Comment (vpereira) [inline deploy_without_migration.yml:15]: doesn't it work checking the program return code or why do we have to parse stdout programatically?

Comment (dmarcoux) [reply deploy_without_migration.yml:15]: At least right now, the return code is always the same (`0`). We don't deal with potential errors: https://github.com/openSUSE/obs_github_deployments/blob/bce55230e102b6eb0f1987b27c2d3415feb9ee22/lib/obs_github_deployments/deployment.rb#L13-L20

Comment (danidoni) [reply deploy_without_migration.yml:15]: Okay, now I see what's the problem. You are right @dmarcoux. I'll fix the playbook and the `obs_github_deployment` to work with exit codes.

Comment (vpereira) [reply deploy_without_migration.yml:15]: there is nothing to change in the playbook...

Comment (danidoni) [reply deploy_without_migration.yml:15]: The return codes are being fixed [here](https://github.com/openSUSE/obs_github_deployments/pull/29). Once that is merged I can remove the stdout parsing.

Comment (danidoni) [reply deploy_without_migration.yml:15]: > there is nothing to change in the playbook...

Reverting the stdout parsing and relying on the return code alone

Comment (vpereira) [reply deploy_without_migration.yml:15]: > Reverting the stdout parsing and relying on the return code alone

Which is what we have today right? So closing this PR would "do the trick", no?https://github.com/openSUSE/ansible-obs/blob/master/deploy_with_migration_without_downtime.yml#L17

Comment (vpereira) [reply deploy_without_migration.yml:15]: https://docs.ansible.com/ansible/latest/user_guide/playbooks_error_handling.html

Comment (danidoni) [reply deploy_without_migration.yml:15]: > > Reverting the stdout parsing and relying on the return code alone
> 
> Which is what we have today right? So closing this PR would "do the trick", no?https://github.com/openSUSE/ansible-obs/blob/master/deploy_with_migration_without_downtime.yml#L17

So true. Closing

**Comment by danidoni** [comment]
Closing because we rely on the return code from the `obs_github_deployments` command line tool instead of this

**Pull request closed without merging**